docs/protocol.md

```diff
@@ -661,15 +661,6 @@ Value is name of the service making the call.
 
 ### Transport Header `re` -- Retry Flags
 
-Flags are encoded as a UTF-8 string, as are all header values. Each flag is
```

Comment:
Maybe a lead in such as: "The value of this field is one or more of the following flags:"